pages/common/kaggle.md

```diff
@@ -0,0 +1,12 @@
+# kaggle
+
+> Official API for kaggle accessible using a command line tool implemented in Python 3.
+> More information: <https://www.kaggle.com>.
```

Comment:
Is any kaggle cli document we can put here? Not just the home page of official site? I've found [this](https://github.com/Kaggle/kaggle-api). 

```suggestion
> More information: <https://github.com/Kaggle/kaggle-api>.
```
I suggest we can put the direct document link here.